Refactor works to be independent of files

mark-dce opened the pull request:
The code had two paths to handle packed files depending on the type of CSV row they appeared in.

This change
* Consolidates handling of packed files in the Tenejo::PFFile class
* Makes `import_path` an instance variable of the graph so we don't have to pass it through intermediate method calls
* Removes a duplicate definition of DEFAULT_UPLOAD_PATH which is also defined [in preflight.rb](https://github.com/curationexperts/tenejo/blob/v2.19.0/app/lib/tenejo/preflight.rb#L9)

little9 commented:
@bess We won't need to run the install command above because the engine code
appends the path to its migrations to the Rails migration path variable. That way when you run
`rails db:migrate` it knows to look in the engine folders for the migrations. That way we only have one set of migrations to mange. 

That is this line: https://github.com/curationexperts/zizia/blob/9be2c8b5c13aa44754c52268e720ce406f40eaef/lib/zizia/engine.rb#L12

mark-dce committed:
Refactor works to be independent of files

The code had two paths to handle packed files depending on the type
of CSV row they appeared in.

This change
* Consolidates handling of packed files in the Tenejo::PFFile class
* Makes `import_path` an instance variable of the graph so we don't
  have to pass it through intermediate method calls
* Removes a duplicate definition of DEFAULT_UPLOAD_PATH which is
  also defined [in preflight.rb](https://github.com/curationexperts/tenejo/blob/v2.19.0/app/lib/tenejo/preflight.rb#L9)
